lib/l10n/app_tr.arb

```diff
@@ -0,0 +1,638 @@
+{
+  "appName": "Aves",
+  "welcomeMessage": "Aves'e Hoş Geldiniz",
+  "welcomeOptional": "İsteğe bağlı",
+  "welcomeTermsToggle": "Hüküm ve koşulları kabul ediyorum",
+  "itemCount": "{count, plural, =1{1 öğe} other{{count} öğe}}",
```

Comment:
Not an issue, but when there are no translation differences between `=1{...}` and `other{...}`, you can omit `=1{...}`.
Here it would give:
```
   "itemCount": "{count, plural, other{{count} öğe}}",
```

Reply:
In Turkish we don't use plurals like that. There is no need for a plural suffix in `
"itemCount": "{count, plural, other{{count} öğe}}",`
It is ok both to be same